weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareEvaluationsPage/TraceCallsCompareEvaluationsPage.tsx

```diff
@@ -0,0 +1,518 @@
+import {Box} from '@material-ui/core';
```

Comment:
I would move this file and the `TraceCallsSection` into a subdir inside of `CompareEvaluationsPage` since it is basically a completely alternative implementation